gulpfile.js

```diff
@@ -23,5 +24,10 @@ gulp.task('copydecs', () => {
 	.pipe(gulp.dest(tsProject.options.declarationDir));
 });
 
-gulp.task('build', [ 'typescript', 'copydecs' ]);
+gulp.task('watch', () => {
+	watch('./lib', () => {
```

Comment:
Would `tsProject.src()` work here? It means that if somebody changes the source directory for whatever reason, the watch functionality will still work.